Katello/katello
# Fixes #33037 - Cancel button should be enabled for capsule sync until job completes

To test:
1. Sync a proxy.
2. Task progress bar should appear with a cancel button.
3. Refresh the page.
4. Task progress bar should appear with Cancel button until job is finished.

Without this change the Cancel button disappears upon refreshing showing the synchronize button even when there's a proxy sync task running in the background.

To test easily, have a big repo and run complete syncs to get a long running task. Or add sleep(30) to `invoke_external_task` of the action pulp3/capsule_content/sync.rb
